src/Pages/Dashboard/FootprintSlideshow/BarChart.js

```diff
@@ -0,0 +1,48 @@
+import React, { useContext } from "react";
+import { VictoryChart, VictoryAxis, VictoryBar, VictoryTheme } from "victory";
+import { UserContext } from "../../../Main/UserContext";
+
+export default () => {
+  const footprint = useContext(UserContext).footprint[0].total.sumTotal;
+  // This holds the data for each user type
+  const data = [
+
+    { user: 1, expenditure: footprint, fill: "#02C39A"}, //This value is the user's value, so it needs to be called somehow from their account details.
+    { user: 2, expenditure: 40, fill: "#C4C4C4" },
+    { user: 3, expenditure: 50, fill: "#C4C4C4" },
+    { user: 4, expenditure: 15, fill: "#C4C4C4" },
+  ];
+
+  return (
+    <div className="slide">
+      {/* Style needs a lot of work */}
+      <div className="barChart">
+        <VictoryChart domainPadding={10} theme={VictoryTheme.material}>
+          <VictoryAxis
+          
```

Comment:
do you use prettier? this is strange formatting.

Reply:
Must have missed one.